gnome-initial-setup/gis-page.h

```diff
@@ -78,7 +78,6 @@ const char *       gis_page_get_forward_text (GisPage *page);
 void         gis_page_set_forward_text (GisPage *page, const char *text);
 gboolean     gis_page_get_complete (GisPage *page);
 void         gis_page_set_complete (GisPage *page, gboolean complete);
-GtkWidget *  gis_page_get_action_widget (GisPage *page);
```

Comment:
Does this need to be sent upstream too?

Reply:
Good point. https://bugzilla.gnome.org/show_bug.cgi?id=793833